src/redis_stack/vector_similarity_search.md

```diff
@@ -133,17 +133,22 @@ HSET bikes:20011  model 'ThrillCycle' brand 'BikeShind' price 815 type 'Commuter
 ```
 
 ```redis Create index for VSS
-FT.CREATE idx:bikes_vss 
+FT.CREATE idx:bikes_vss
     ON HASH
         PREFIX 1 "bikes:"
- SCHEMA 
-   model TEXT NOSTEM SORTABLE 
-   brand TEXT NOSTEM SORTABLE
-   price NUMERIC SORTABLE 
-   type TAG 
-   material TAG
-   weight NUMERIC SORTABLE    
-   description_embeddings VECTOR FLAT 6 TYPE FLOAT32 DIM 768 DISTANCE_METRIC COSINE
+ SCHEMA
+   "model" TEXT NOSTEM SORTABLE
+   "brand" TEXT NOSTEM SORTABLE
+   "price" NUMERIC SORTABLE
+   "type" TAG
+   "material" TAG
+   "weight" NUMERIC SORTABLE
+   "description_embeddings" VECTOR "FLAT" 10
+        "TYPE" FLOAT32
+        "DIM" 768
+        "DISTANCE_METRIC" "L2"
+        "INITIAL_CAP" 20
```

Comment:
Are there only 20 bikes in the total in the db? Set the INITIAL_CAP and BLOCK_SIZE to the total number of bike hashes in the db....